docs/data/charts/tooltip/ItemTooltipFixedY.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import NoSsr from '@mui/material/NoSsr';
+import Popper from '@mui/material/Popper';
+import { useItemTooltip, useMouseTracker } from '@mui/x-charts/ChartsTooltip';
+import { useDrawingArea, useSvgRef } from '@mui/x-charts/hooks';
+import { CustomItemTooltipContent } from './CustomItemTooltipContent';
+import { generateVirtualElement, MousePosition } from './generateVirtualElement';
+
+export function ItemTooltipFixedY() {
+  const tooltipData = useItemTooltip();
+  const mousePosition = useMouseTracker();
+  const svgRef = useSvgRef(); // Get the ref of the <svg/> component.
+  const drawingArea = useDrawingArea(); // Get the dimensions of the chart inside the <svg/>.
+
+  if (!tooltipData || !mousePosition) {
+    // No data to display
+    return null;
+  }
+
+  const tooltipPosition: MousePosition = {
+    ...mousePosition,
+    // Add the y-coordinate of the <svg/> to the to margin between the <svg/> and the drawing area
+    y: svgRef.current.getBoundingClientRect().top + drawingArea.top,
```

Comment:
Is there any way we can improve on `svgRef.current.getBoundingClientRect().top`?

Maybe we could have a `useDrawingArea().screenCoodinate.top`? 

Or a standalone hook, as `getBoundingClientRect` is costly. Eg: `useDrawingAreaScreenCoordinate().top`

Reply:
Would it be a hook that would update a state by calling the `getBoundingClientRect` when receiving an event about

- a resize of the SVG
- a scroll of the window

Could be interesting but more in an followup PR